src/components/cards/CardItem.vue

```diff
@@ -179,6 +188,8 @@ export default {
 				flex-grow: 1;
 				font-size: 100%;
 				cursor: text;
+				overflow-x: hidden;
+				text-overflow: ellipsis;
```

Comment:
```suggestion
				word-wrap: break-word;
```

I would rather break the words instead of ellipsis

Reply:
Sounds good. Done.